pkg/products/marin3r/apiUsagePrometheusRules.go

```diff
@@ -16,7 +16,10 @@ import (
 )
 
 var (
-	totalRequestsMetric = "ratelimit_service_rate_limit_apicast_ratelimit_generic_key_slowpath_total_hits"
+	totalRequestsMetric = []string{
+		"authorized_calls",
+		"limited_calls",
+	}
```

Comment:
Used in API Usage alerts and using `authorized_calls` and `limited_calls` as the previous metrics was a total of both but should we just be using `authorized_calls` ? :thinking: